.circleci/config.yml

```diff
@@ -287,12 +287,12 @@ commands:
           name: Build Docker image
           command: |
             echo Using SHA $(cat tmp/gitsha | head -c7)
-            docker build -t << pipeline.parameters.ecr-repo >>:$(cat tmp/gitsha | head -c7) .
+            docker build -t << pipeline.parameters.ecr-repo >>:sha-$(cat tmp/gitsha | head -c7) .
```

Comment:
helm deployment fix...  if the SHA starts with a number, helm makes is exponential 2.4345e^6 which is not a valid deployment or image tag name.